### ISSUE TITLE
SQL Populate Scripts no longer include table create statements

### ISSUE THREAD

**Issue by HeinousK** [JD-Software/JDeSurvey#6]
I tried to create the demo_survey schema as described in the video (after failing to find anyplace that described the schema to begin with, as I had tried to run the populate scripts and had it fail), and noticed in the video that the version of the populate script used for the video is distinctly different than what is currently on the site.  The site only contains an update version - assuming the tables are already there and defined.  It does not have the initial table create instructions.  While I could reverse engineer it, I would greatly appreciate not having to.  I recognize this has been dormant for some time, but am hoping someone can still provide the full SQL script.

**Comment by harishchander67** [comment]
Having the same issue. Was there any input?

**Comment by HeinousK** [comment]
No, I didn't receive any input.  I did; however, determine how to proceed.  Their documentation was based on having to create the tables first.  In reality, since they are relying on the hibernate libraries, all that you really need to do is to create the database, and make sure you have configured the database connection information.  After compiling, when you deploy the war files to TomCat (or other), it will create all of the tables that it requires as it goes through the initialization of the war file.  After the war files are fully deployed, you can then run the provided SQL scripts, to get sample data.

By the way, if you haven't been able to successfully compile - there is a dependency referenced in three of the projects that needs to be updated.  The artifactId 'com.springsource.javax.validation' needs to be changed to 'validation-api' in the POM files.  It should then compile.

After compiling, and trying to run - I had an internal error when trying to create a survey.  It turns out that it is failing to find a properties string.  The property it is looking for is 'isAutoReminderEnabled_tip', and you'll find the valid value in the jdsurvey-core\src\test\ directory structure, but not in a non-test location.  I placed it in jdsurvey-intweb\src\main\webapp\WEB-INF\il8n\messages.properties - but wonder if it would have been more appropriate in jdsurvey-core?  I put it where I did, because that's where the error was being reported - and I didn't know (and still don't) enough about how the project is architected to be sure if it would pick it up from core, and so I've never tested it.

Since getting it working, I've spent most of my time trying to get a Radio Button Matrix control working with it.  It seemed like it would be easy to add, but I have not found that to be the case.  I think I'm almost done, but during this process, I've come across SurveyJS and have wondered if it wouldn't have made more sense spending some time doing some framework development around that.  It has a lot of Question Types, and all of it is JSON, and so should be easy to store the data.  At least, it seems to still be actively being developed.  It's a real shame no one chose to maintain this application, as it certainly has (or had) potential.   

Best of luck!
